src/services/authService.js

```diff
@@ -17,46 +17,46 @@ export function SignUp(NameFirst, NameLast, Email, Password, Telephone, UniqueId
 		SuppressNotifications });
 }
 export function ResetPassword(Email) {
-	const endpointResetPassword = returnAPIEndPoint("apiPasswordResetUrl");
+	const endpointResetPassword = returnConfigItems("endPoints","apiPasswordResetUrl");
 	return http.post(endpointResetPassword, { Email });
 }
 export function CreateReport(data) {
-	const endpointReport = returnAPIEndPoint("apiReportUrl");
+	const endpointReport = returnConfigItems("endPoints","apiReportUrl");
 	return http.post(endpointReport, data );
 }
 
 export function GetReportByID(ReportID) {
-	const endpointReport = returnAPIEndPoint("apiReportUrl");
+	const endpointReport =returnConfigItems("endPoints","apiReportUrl");
 	return http.get(endpointReport + ReportID);
 }
 
 export function GetReportByLatLong(X, Y) {
-	const endpointReport = returnAPIEndPoint("apiReportUrl");
+	const endpointReport = returnConfigItems("endPoints","apiReportUrl");
```

Comment:
similar comment to above. any config items that are repeated can be placed in a variable above.

Example 

```js
returnConfigItems("endPoints","apiReportUrl");
```

Reply:
fixed